packages/devel/libffi/package.mk

```diff
@@ -30,7 +30,7 @@ PKG_LONGDESC="The libffi library provides a portable, high level programming int
 PKG_TOOLCHAIN="autotools"
 
 PKG_CONFIGURE_OPTS_TARGET="--disable-debug \
-                           --enable-static --disable-shared \
+                           --enable-static --enable-shared \
```

Comment:
This is needed until Amlogic delivers GBM only libMali for 32bit... we have the correct binary for 64bit already.